chainbench/util/event.py

```diff
@@ -26,39 +47,50 @@ def on_init(environment, **_kwargs):
 
     host_under_test = environment.host or "Default host"
 
-    if isinstance(environment.runner, MasterRunner):
+    if not isinstance(environment.runner, MasterRunner):
```

Comment:
Why using `not` here?

Reply:
This is following the locust code examples. I believe it is so that the same code will still work for local runner, when we don't run in distributed mode (master, worker).